src/main/resources/stubs/time/time.gobra

```diff
@@ -238,7 +238,6 @@ ensures forall i int :: 0 <= i && i < len(data) ==> acc(&data[i], p)
 func (t *Time) UnmarshalBinary(data []byte, ghost p perm) error
 
 // GobEncode implements the gob.GobEncoder interface.
-requires forall i int :: 0 <= i && i < len(res) ==> acc(&res[i])
 ensures forall i int :: 0 <= i && i < len(res) ==> acc(&res[i])
 func (t Time) GobEncode() (res []byte, error)
```

Comment:
Out of curiosity: why are some methods defined on `Time` and others on `*Time`?

Reply:
I believe that is a common idiom in Go. When you want to access the fields of a receiver `t` but you don't want to modify them, you use `Time` and this creates a copy of the structure, otherwise you pass `t` as a `*Time`.

Reply:
True, this is a nice way to express immutability, but is Go clever enough to not copy the struct at runtime? Otherwise, it seems to be quite a high price just to express that something is immutable

Reply:
I don't know about that :/ However, I do not think we should be concerned about it. In fact, this is the way that it is implemented in the Gobra std library

Reply:
I would however assume that Go indeed makes a copy of the structure, at least when you modify the fields of the struct (for some reason, you may want to modify the argument in the scope of your method without modifying the original value that was passed)